src/app/policy-update/policy-update.component.html

```diff
@@ -0,0 +1,26 @@
+<!--
+   Copyright 2019 OICR
+ *
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ *
+       http://www.apache.org/licenses/LICENSE-2.0
+ *
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<app-header>
+  Policy Agreement Required
+</app-header>
+<div class="container">
+  <div class="column">
+    <p class="lead">
+      You have been logged out because it looks like you need to agree to our latest Terms of Service and Privacy Policy. Please
```

Comment:
Sort of torn on "looks like". On the one hand it's more polite. On the other hand, it may seem like we don't know. :)

Reply:
Oh yeah, I was going for polite sounding but I can see the other point of view now that you mention it. If others agree for a firmer message, I will change it.

Reply:
I feel like this should be more generic like "You session has ended" or "You have been signed out because your session has expired"

Reply:
> I feel like this should be more generic like "You session has ended" or "You have been signed out because your session has expired"

I think we should make it explicit to the user, but maybe that's overkill since the log-in page has the passive acceptance.